tests/apps/stateapp/app.go

```diff
@@ -353,30 +414,54 @@ func httpHandler(w http.ResponseWriter, r *http.Request) {
 	statestore := mux.Vars(r)["statestore"]
 	switch cmd {
 	case "save":
-		err = save(req.States, statestore)
+		if req, err = parseRequestBody(w, r); err != nil {
+			return
```

Comment:
Why err is being ignored now and not handled after the switch statement?

Reply:
the error is handled inside `parseRequestBody`